### ISSUE TITLE
Can't use certain resource packs

### ISSUE THREAD

**Issue by SomeFafr** [bluemagic123/tModLoader#2706]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

I just encountered a weird thing - when I try to use resource packs, I can't exit the window when certain resource packs are used, why does that happen? When some packs work individually, they can't work together for some reason
I noticed that when I press something and it doesn't work, the window is shown in 1024x768 resolution for a tiny moment
It doesn't seem to be influenced by the pack's size, but something else. I have a texture pack that only changes the UI, and it didn't work. But when I removed all the sprites except worldgen ones it worked. But it makes no sense because this pack, before removing most sprites, was much smaller in every single way than another pack that always works.
All mods are off, still happens
Both local and steam packs work, but some packs from either of those two places don't.

### Steps to reproduce

1. Go to the "Use Resource Packs" tab
2. Start using the ones that don't work
3. When pressing "back", the screen flickers between 1024x768 and back to 1920x1080, and nothing happens

### Expected Behavior

I should be sent to the Workshop Hub tab, with the packs working

### Actual Behavior

The screen flickers to 1024x768 and back to 1920x1080, nothing else happens

### Reproduction frequency

Every time, as long as the certain not working packs are active

### Additional Information

I use Windows 7, some of the XNA files on my PC are corrupted, it doesn't depend on the pack size (a 20mb pack can load, while a 10mb one can't), doesn't seem like others had this issue, others had similar issues, but not the same ones

**Comment by JavidPack** [comment]
You didn't post the client.log file

**Comment by SomeFafr** [comment]
> You didn't post the client.log file

Thanks for saying this, I feel really stupid now. Turns out that unlike vanilla Terraria, TModLoader tries to use every file with the same name, as opposed to using a file with a certain format in Vanilla. I had multiple variations of the same thing in different formats, and I used that to modify the things myself. For example: I am currently working on a joke resource pack that oversimplifies the entire UI, and to mark the WIP sprites I place them next to the .png file, but the WIP is the .pdn file. TML gets confused and says that a file his found with multiple extensions, and after removing the .pdn file, it works perfectly.
Again, thanks for saying this, I feel stupid for not realizing that logs exist, but at the same time I have a feeling of accomplishment because I fixed a problem myself for the first time.
